### PR TITLE
JBPM-6294 - Avoid null or empty values in dropdown

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java [hunk 1]
```diff
@@ -40,10 +42,14 @@
 import static org.junit.Assert.*;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.doAnswer;
 
 @RunWith(MockitoJUnitRunner.class)
 public class DataSetAwareSelectTest {
 
+    private static final String VALUE_COLUMN_ID = "value";
+    private static final String TEXT_COLUMN_ID = "test";
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java]: Is here "test" put intentionally or it's a typo and should be "text" as in ```TEXT_COLUMN_ID = "text"```?

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java [hunk 2]
```diff
@@ -113,18 +124,75 @@ public void testLookupDataSet() throws Exception {
...
+        assertEquals(dataUUID,
+                     cdsl.getDataSetUUID());
+        verify(select).addOption(columnText,
+                                 columnValue);
+        verify(select).enable();
+        verify(select).removeAllOptions();
+    }
+
+    @Test
+    public void testLookupDataSetEmptyString() throws Exception {
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java]: Would only suggest renaming the method name to ```testLookupDataSetEmptyStringOrNull()```

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java]: Fixed.

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java (hunk 2 above)
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java]: Done

**Pull request merged**